python/ray/serve/api.py

```diff
@@ -112,18 +112,22 @@ def async_loop(self):
         # called by handles
         return self._async_loop
 
-    def _remote(self, endpoint_name, handle_options, request_data,
-                kwargs) -> Coroutine:
+    def _remote(self,
+                endpoint_name,
+                handle_options,
+                args,
+                kwargs,
+                use_serve_request=True) -> Coroutine:
```

Comment:
```suggestion
                use_serve_request=True,) -> Coroutine:
```
and run the yapf formatter, it will make the format much nicer. trailing-comma >> no trailing